src/style/reset.css

```
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
```

You can write all params here
```suggestion
body,
h2,
p {
  margin: 0;
  padding: 0;
}
```